ansible/examples/slave.yml

```diff
@@ -158,6 +158,24 @@
         line: 'enabled=0'
       when: ansible_os_family == "RedHat"
 
+    # This repo has EL8 packages that Ceph depends on.  These packages are not available in AppStream, BaseOS, or EPEL8 yet so we build and manage them.
+    - name: Enable EL8 copr repo
+      command: "dnf -y copr enable ktdreyer/ceph-el8"
+      when:
+        - ansible_os_family == "RedHat"
+        - ansible_distribution_major_version|int >= 8
```

Comment:
This should not be "el8 or greater" since this repo is specifically for 8.

`>=` should be `==`